### PR TITLE
Add Incision Management System upgrade for medical cyborgs

### PR DESCRIPTION
**Pull request by PIerreLouisH**
## What Does This PR Do
Add Incision Management System upgrade for medical cyborgs, it'll replace the standard cyborg scapel.

## Why It's Good For The Game
As a doc i always get the IMS whenever available at RnD, as it make the surgeries way smoother.
The few times i play as a medical borg i end-up frustrated for one thing: not being able to have the upgrade even with high levels researched. 
It's still on the logic of adding a few more options to borgs, while not changing the balance of the game whatsover.

## Images of changes
image

## Testing
Spawned as a medical cyborg and as roboticist, put the upgrade on it. Everything worked.

## Changelog
:cl:
add: Add Incision Management System upgrade for medical cyborgs
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
remove empty line from changelog

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -262,6 +262,16 @@
 	for(var/obj/item/reagent_containers/borghypo/F in R.module.modules)
 		F.emag_act()
 
+/obj/item/borg/upgrade/incision_management
+	name = "medical cyborg IMS upgrade"
+	desc = "An Incision Managment System replacment for the medical cyborg's standard scapel."
```
Comment (Henri215) [inline code/game/objects/items/robot/robot_upgrades.dm:267]:
```suggestion
	desc = "An Incision Management System replacment for the medical cyborg's standard scapel."
```

Comment (PIerreLouisH) [reply code/game/objects/items/robot/robot_upgrades.dm:267]: Ah thank you for picking it up! English my beloved.

**Comment by 1080pCat** [comment]
What is the borg going to do if they need to do a surgery that has a scalpel as a step in it? like facial reconstruction, or the burn/dead tissue surgery? the IMS isn't able todo those.

Also, while im very much for borg buffs, i really dont see the need for med borgs getting a IMS, it'd be better to give them a surgical speed upgrade, to make the tools the have faster.  Thats just my opinion though.

**Comment by PIerreLouisH** [comment]
> What is the borg going to do if they need to do a surgery that has a scalpel as a step in it? like facial reconstruction, or the burn/dead tissue surgery? the IMS isn't able todo those.
> 
> Also, while im very much for borg buffs, i really dont see the need for med borgs getting a IMS, it'd be better to give them a surgical speed upgrade, to make the tools the have faster. Thats just my opinion though.

You are right i forgot about the few case where only a normal scapel would work.

Hence i could just add the IMS to the list of med borgs.

Having said that, since it's deemed useless for a medborg to have IMS in the first place, i'd rather perma close the PR.

I personnally am not in favour to give - even medborg - a bonus speed that standard humans can't match.
Maybe with locked rareish illegal levels? Even so it'll be another PR, and not one i'd like to make, as it would change the balance of the game, even if slightly.

**Pull request closed without merging**